conf.mk

```diff
@@ -29,5 +29,5 @@ COCO_GCCRETRO_URL:="https://ftp.gnu.org/gnu/gcc/$(COCO_GCCRETRO_VERSION)/$(COCO_
 
 EOU_H6309_URL:=http://www.lcurtisboyle.com/nitros9/EOU_Version%201_0_0_(6309_ONLY)_12-02-2022.zip
 EOU_M6809_URL:=http://www.lcurtisboyle.com/nitros9/EOU_Version%201_0_0_(6809_ONLY)_12-02-2022.zip
-EOU_H6309_ZIP:=EOU_Version?1_0_0_?6309_ONLY?_12-02-2022.zip
-EOU_M6809_ZIP:=EOU_Version?1_0_0_?6809_ONLY?_12-02-2022.zip
```

Comment:
this change is not going to work with wget.  But we have `curl -O` on Ubuntu, so use that for all platforms, and this change will be OK.